front/components/spaces/ContentActions.tsx

```diff
@@ -137,11 +137,8 @@ export const ContentActions = React.forwardRef<
 
     return (
       <>
-        {currentAction.action === "TableUploadOrEdit" ? (
-          <TableUploadOrEditModal {...modalProps} />
-        ) : (
-          <DocumentUploadOrEditModal {...modalProps} />
-        )}
+        <TableUploadOrEditModal {...modalProps} />
```

Comment:
Had to ditch the conditional rendering other modals would be unmounted -> breaking the transition